### PR TITLE
fix(multi-platform): add buildx and multiplatform support

### LINKED ISSUE #5: Fundamentals tutorial "container exited immediately"
### What happened?

Following the [Pulumi introduction tutorial](https://www.pulumi.com/learn/pulumi-fundamentals) I can't get the app running with the default MongoDB image.

### Steps to reproduce

After adding the containers step and executing `pulumi up` I get the following error:
```
docker:index:Container (mongo_container):
  error: 1 error occurred:
   * container exited immediately
```

By executing locally the container:

```
docker run -dit pulumi/tutorial-pulumi-fundamentals-database-local
```

I notice is not using the same image platform as the `backend` and `frontend` images:
```
WARNING: The requested image's platform (linux/arm64/v8) does not match the detected host platform (linux/amd64) and no specific platform was requested
```

### Expected Behavior

If building the image from this repo I can complete the tutorial but it would be great to update the docker hub image

### Actual Behavior

I can't complete the tutorial following the instructions

### Versions used

CLI          
Version      3.33.2
Go Version   go1.17.10
Go Compiler  gc

Host     
OS       ubuntu
Version  22.04
Arch     x86_64

Backend        
Name           pulumi.com

### Additional context

_No response_

### Contributing

Vote on this issue by adding a 👍 reaction. 
To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already).

### PR DESCRIPTION
**Pull request by nimbinatus**
fixes #5, https://github.com/pulumi/pulumi-hugo/pull/1733

### CODE REVIEW

**Review by scottslowe — CHANGES_REQUESTED** [review]
See my comments

File: .github/workflows/publish.yaml [hunk 1]
```diff
@@ -32,11 +32,18 @@ jobs:
           images: ${{ env.REGISTRY }}/${{ env.FRONT_IMAGE_NAME }}
           flavor: |
             latest=true
+            
+      - name: Set up QEMU
+        uses: docker/setup-quemu-action@v2
```
Comment (dirien) [inline .github/workflows/publish.yaml:37]:
```suggestion
        uses: docker/setup-qemu-action@v2
```

**Review by dirien — CHANGES_REQUESTED** [review]
Just a typo in the action!

File: .github/workflows/publish.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (nimbinatus) [reply .github/workflows/publish.yaml:37]: 🤦‍♀️ yes; that's what I get for typing in a hurry

File: .github/workflows/publish.yaml [hunk 2]
```diff
@@ -90,11 +104,18 @@ jobs:
           images: ${{ env.REGISTRY }}/${{ env.DATABASE_IMAGE_NAME }}
           flavor: |
             latest=true
+            
+      - name: Set up QEMU
+        uses: docker/setup-quemu-action@v2
```
[the remark this answers is not in the document]
Comment (nimbinatus) [reply .github/workflows/publish.yaml:109]: 🤦‍♀️ that's what I get for typing in a hurry. Thanks!

**Pull request merged**